fmf/base.py

```diff
@@ -191,6 +191,44 @@ def _merge_plus(self, data, key, value, prepend=False):
                 "MergeError: Key '{0}' in {1} ({2}).".format(
                     key, self.name, str(error)))
 
+    def _merge_regexp(self, data, key, value):
+        """ Handle substitution of current values """
+        if isinstance(value, str):
+            value = [value]
+        for pattern, repl in [utils.split_pattern_repl(v) for v in value]:
+            if isinstance(data[key], list):
+                data[key] = [re.sub(pattern, repl, original) for original in data[key]]
```

Comment:
Shouldn't there be a check or try catch that `original` is a `str`?

Reply:
fixed in 5de1c69